hack/validate-imports/allowed-import-names.yaml

```diff
@@ -109,3 +109,5 @@ allowedImportNames:
     - hivefake
   github.com/openshift/hive/apis/hive/v1:
     - hivev1
+  github.com/gofrs/uuid:
+    - gofrsuuid
```

Comment:
am I the only one who reads this as "gophers' uuid"?

Reply:
I think that 'gofrs' is meant to invoke that, yes :)